src/components/sections/TicketsSection.tsx

```diff
@@ -6,11 +6,11 @@ export const TicketsSection = () => {
   return (
     <Section id="vstupenky" className="">
       <div className="max-w-7xl mx-auto">
-        <h2 className="text-2xl sm:text-3xl font-semibold uppercase font-numbers mb-4">
+        <h2 className="text-2xl sm:text-3xl font-semibold uppercase font-numbers mb-4 mx-2">
```

Comment:
do we need this mx? I think it was aligned already, now it would be a bit off and not aligned with the images down there